src/transactions/send.ts

```diff
@@ -323,7 +323,35 @@ export function* wrapSendTransactionWithRetry(
   }
 }
 
-function shouldTxFailureRetry(err: any) {
+/**
+ * Given an error thrown while attempting a transaction,
+ * checks to see if that error could indicate that the transaction intent already
+ * exists on the blockchain.
+ **/
+export function isTxPossiblyPending(err: any): boolean {
```

Comment:
These checks were pulled out, since the other cases within `shouldTxFailureRetry` do not indicate that the TX may potentially be in progress on the blockchain.